gui/slick/css/style.css

```diff
@@ -1894,6 +1899,17 @@ span.quality {
 	border-radius: 4px;
 }
 
+span.any-hd {
+	background-color: #2672b6;
+	background: repeating-linear-gradient(
```

Comment:
According to [Can I use](http://caniuse.com/#feat=css-repeating-gradients), one browser still needs the `-webkit-` prefix here.